app/controllers/further_education_payments/providers/claims/verifications_controller.rb

```diff
@@ -32,7 +32,7 @@ def update
             elsif wizard.completed?
               flash[:success] = "Claim Verified for #{@form.claimant_name}"
 
-              redirect_to further_education_payments_providers_claims_path
+              redirect_to further_education_payments_providers_verified_claims_path
```

Comment:
Do we want to update the `redirect_if_verified` callback to send them to the verified claims show action?